README.md

```diff
@@ -167,3 +167,14 @@ Example steps taken by the function-to-app converter may include:
 ## Stdout/Stderr and Logging expectations
 
 Application logs to stdout/stderr within application code and logs from the Functions Framework itself are expected to appear in stdout/stderr of the process running the Functions Framework.
+
+## HTTP Status Codes
+
+The framework should return 4XX HTTP status error codes for client errors and 5XX HTTP status error codes for Function Framework errors. Examples:
+
+- 4XX: Client errors
```

Comment:
I would expect the code labels for these to use the RFC specs ([standard](https://tools.ietf.org/html/rfc7231#page-47), and [more for 429](https://tools.ietf.org/html/rfc6585#section-4), ...) then add FF-specific descriptions for specific context.

Reply:
Good suggestion. Added!